### ISSUE TITLE
CMake bug on develop branch

### ISSUE THREAD

**Issue by ksiero** [leobago/fti#326]
There is a CMake bug on `develop` branch in https://github.com/leobago/fti/blob/develop/CMakeLists.txt#L264. 
`${fti_target_sources}` is not a target, this is a list of source files.

`${fti_target_sources}` should be replaced by `${fti_targets}`.

**Comment by ksiero** [comment]
I tried to fix it on my own, but still I am getting an error:
```
-- The C compiler identification is GNU 9.3.0
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Found ZLIB: /usr/lib/x86_64-linux-gnu/libz.so (found version "1.2.11") 
-- Found MPI: /usr/bin/mpicc
      
-- Could NOT find OpenSSL, try to
   set the path to OpenSSL root folder in the system variable OPENSSL_ROOT_DIR (missing:  OPENSSL_SSL_LIBRARY OPENSSL_CRYPTO_LIBRARY OPENSSL_INCLUDE_DIR)
         
-- Using slower build in MD5 algorithm.
Libraries: 
Libraries: 
CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
Please set them or make sure they are set and tested correctly in the CMake files:
OPENSSL_CRYPTO_LIBRARY (ADVANCED)
    linked by target "fti.shared" in directory /home/sierot/Workspace/pdi/build/FTI/src/FTI_pkg
    linked by target "fti.static" in directory /home/sierot/Workspace/pdi/build/FTI/src/FTI_pkg
OPENSSL_SSL_LIBRARY (ADVANCED)
    linked by target "fti.shared" in directory /home/sierot/Workspace/pdi/build/FTI/src/FTI_pkg
    linked by target "fti.static" in directory /home/sierot/Workspace/pdi/build/FTI/src/FTI_pkg
```

**Comment by ksiero** [comment]
Managed to fixed it in #332

**Comment by leobago** [comment]
Thank you @ksiero, the fix has been integrated into the master branch, as well as the new tag 1.4.1.
